scripts/release.sh

```diff
@@ -133,6 +133,25 @@ git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
 echo "Deploying release to maven central"
 mvn deploy -Prelease -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" -DprocessAllModules=true ${MVN_DEPLOY_DRYRUN:-}
 
+echo "Creating binary distribution"
+mvn package -Pdist -DskipTests=true
```

Comment:
Should we be signing the release artefacts? Should we also be creating md5/sha1 checksums to with the binaries? 

We can do that in a follow up PR if preferred.

Reply:
We probably should, but here I was going for the bare minimum of having a binary artefact attached to the release at all 😉

Reply:
Yeah stick something in issues.

Reply:
Agreed that it probably should be in Maven too, but that was beyond the scope of what I was trying to achieve with this PR. Here I was simply trying to ensure we are making a binary distribution available in the repository with each release so it can be referenced in the quick start guide, so that new users don't have to build the whole project from source in order to install it.